node_modules/gh-orgunit/lib/api.js

```diff
@@ -968,3 +968,49 @@ var unsubscribeOrgUnit = module.exports.unsubscribeOrgUnit = function(ctx, id, u
         });
     });
 };
+
+/* Export / Import */
+
+/**
+ * Export the organisational units, event series and events under an organisational unit
```

Comment:
Can you clarify if this will walk the tree and also export the organisational units, event series and events of the organisational units under the provided organisation unit ?